Tools/Scripts/libraries/webkitscmpy/webkitscmpy/test/pull_request_unittest.py

```diff
@@ -329,7 +329,8 @@ def test_github(self):
         self.assertEqual(
             captured.stdout.getvalue(),
             "Created the local development branch 'eng/pr-branch'!\n"
-            "Created 'PR 1 | Created commit'!\n",
+            "Created 'PR 1 | Created commit'!\n"
```

Comment:
Why are we printing 'Created commit'? 
or is it the title of the PR in this case. If so, let's change the title to something else like " Sample PR".

Reply:
It's the title of the PR, because the title of the PR is the first line of the commit message (which is `Created commit`, in this case)

Reply:
ok. Can we change that to something like: "Created sample commit", to make it clear that "Created commit" is not a literal string which would be there in the actual run.

Reply:
I made it look like an actual bug title, it's `[Testing] Creating commits` and `[Testing] Amending commits` now